planning/behavior_path_planner/include/behavior_path_planner/data_manager.hpp

```diff
@@ -24,6 +24,7 @@
 #include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
 #include <geometry_msgs/msg/accel_with_covariance_stamped.hpp>
 #include <geometry_msgs/msg/pose_stamped.hpp>
+#include <nav_msgs/msg/occupancy_grid.hpp>
```

Comment:
Is this need?

Reply:
Yes, if I remove this line, it cannot build successfully

Reply:
But can following code  be remove? then, occupancy_grid.hpp is no need?
https://github.com/autowarefoundation/autoware.universe/pull/2472/files/b23c71b7df33e7c4f2edbb035433b5aa96e76b18#diff-1f3c3eb13ecfc5dc35e5a803c686d92a0509fddd6525c8d174af256dbab5c681R41
https://github.com/autowarefoundation/autoware.universe/pull/2472/files/b23c71b7df33e7c4f2edbb035433b5aa96e76b18#diff-1f3c3eb13ecfc5dc35e5a803c686d92a0509fddd6525c8d174af256dbab5c681R76

Reply:
@yukkysaito I think pull over uses occupancy grid, which is made in perception module, we cannot omit this line

Reply:
I see :+1: